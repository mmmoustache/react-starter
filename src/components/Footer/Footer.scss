@import '../../styles/settings.scss';
@import '../../styles/tools.scss';

.root {
  a {
    text-decoration: none;

    &:hover,
    &:active,
    &:focus {
      text-decoration: underline;
    }
  }
}
