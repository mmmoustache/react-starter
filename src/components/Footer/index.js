import React from 'react';
import s from './Footer.scss';
import { Row, Column } from 'react-foundation';

class Footer extends React.Component {
  render() {
    return (
			<footer className={s.root}>
        <Row>
          <Column small={12} className="text-center">
            <p>Made with love ‚ù§</p>
          </Column>
        </Row>
      </footer>
		);
	}
}

export default Footer;
