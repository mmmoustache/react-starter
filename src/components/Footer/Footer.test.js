import React from 'react';
import renderer from 'react-test-renderer';
import Footer from '../Footer';

describe('Component: ContentWrapper', () => {

  it('Should render successfully', () => {
    const component = renderer.create(
      <Footer />
    );

    expect(component.toJSON()).toMatchSnapshot();
  });

});