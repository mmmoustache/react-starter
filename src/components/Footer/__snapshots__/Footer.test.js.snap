// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: ContentWrapper Should render successfully 1`] = `
<footer
  className={undefined}
>
  <div
    className="row"
  >
    <div
      className="columns text-center small-12"
    >
      <p>
        Made with love ‚ù§
      </p>
    </div>
  </div>
</footer>
`;
